common/src/Microsoft.Azure.IIoT.Hub.Processor/src/Services/EventProcessorFactory.cs

```diff
@@ -308,7 +330,17 @@ private class EventProperties : IDictionary<string, string> {
             private static readonly Gauge kEventProcessorDetails = Metrics
                 .CreateGauge("iiot_event_processor_info", "details about event processor",
                     new GaugeConfiguration {
-                        LabelNames = new[] { "id", "partition_id", "status" }
+                        LabelNames = new[] { "id", "eventhub_name", "consumer_name", "partition_id", "status" }
+                    });
+            private static readonly Gauge kOldEvent = Metrics
+                .CreateGauge("iiot_event_processor_old_events", "number of messages skipped due to being old",
```

Comment:
I think that `iiot_event_processor_skipped_events` is a better name, as even the description uses that.

Reply:
Please use either term `events` or `messages` for `iiot_event_processor_old_events` and `iiot_event_processor_message_count`. Having both implies that there is a difference.

Reply:
done